webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -237,46 +256,52 @@
             </a>
           </span>
 
-          <a *ngIf="actionBar?.right?.sendTo"
-             mmAuth="can_view_call_action"
-             class="mm-icon mm-icon-inverse mm-icon-caption mobile-only"
-             [attr.href]="'tel:' + actionBar?.right?.sendTo?.phone | safeHtml:'url'"
-             [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
+          <a
+            *ngIf="actionBar?.right?.sendTo"
+            mmAuth="can_view_call_action"
+            class="mm-icon mm-icon-inverse mm-icon-caption mobile-only"
+            [attr.href]="'tel:' + actionBar?.right?.sendTo?.phone | safeHtml:'url'"
+            [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
           >
             <span class="fa fa-phone"></span>
             <p>{{'call' | translate}}</p>
           </a>
-          <a *ngIf="actionBar?.right?.sendTo"
-             mmAuth="can_view_message_action"
-             class="mm-icon mm-icon-inverse mm-icon-caption mobile-only"
-             [attr.href]="'sms:' + actionBar?.right?.sendTo?.phone | safeHtml:'url'"
-             [ngClass]="{ 'mm-icon-disabled': !actionBar?.right?.sendTo?.phone }"
+          <a
+            *ngIf="actionBar?.right?.sendTo"
+            mmAuth="can_view_message_action"
```

Comment:
I think this should be "can_edit" too?

```suggestion
            mmAuth="can_edit,can_view_message_action"
```

Reply:
This action is for mobile device, it opens a sms app using the link `sms:+12345667854`, it's not writing the data base. We discussed this in the stand up last week, even if we block it, ppl can always sms someone directly using their default sms app.

Reply:
Ahh right, sorry! I thought this was a CHT message, not an SMS message button!